spec/integration/guest_view_spec.rb

```diff
@@ -47,6 +45,14 @@
       expect(page).to have_content("milk")
     end
 
+    it "shows the loaded image for each item" do
+      create_item
+
+      visit menu_path
+
+      expect(page.find('.menu_list_img')['src']).to have_content('heart_pizza.gif')
```

Comment:
Line is too long. [83/80]<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.